EXILED/Exiled.API/Features/Items/Firearm.cs

```diff
@@ -592,20 +605,22 @@ public void ClearPreferences()
         /// <param name="emptyMagazine">Whether empty magazine should be loaded.</param>
         public void Reload(bool emptyMagazine = false)
         {
-            MagazineModule magazineModule = Base.Modules.OfType<MagazineModule>().FirstOrDefault();
-
-            if (magazineModule == null)
-                return;
-
-            magazineModule.ServerRemoveMagazine();
+            if (ammoModule is MagazineModule magazineModule)
+            {
+                magazineModule.ServerRemoveMagazine();
 
-            Timing.CallDelayed(0.1f, () =>
+                Timing.CallDelayed(0.1f, () =>
+                {
+                    if (emptyMagazine)
+                        magazineModule.ServerInsertEmptyMagazine();
+                    else
+                        magazineModule.ServerInsertMagazine();
+                });
+            }
+            else if (Base.TryGetModule(out AnimatorReloaderModuleBase animatorModule))
             {
-                if (emptyMagazine)
-                    magazineModule.ServerInsertEmptyMagazine();
-                else
-                    magazineModule.ServerInsertMagazine();
-            });
+                animatorModule.StartReloading();
+            }
```

Comment:
I don't get these changes, isn't StartReloading method client-sided? (it's only used by `ClientProcessRpcInstance`)

Reply:
Nvm, I didn't look at the method correctly